airflow/secrets/__init__.py

```diff
@@ -16,11 +16,13 @@
 # specific language governing permissions and limitations
 # under the License.
 """
-Secrets framework provides means of getting connection objects from various sources, e.g. the following:
+Secrets framework provides means of getting connection objects from various sources.
 
-    * Environment variables
-    * Metastore database
-    * AWS SSM Parameter store
+The following sources are available:
+
+* Environment variables
+* Metastore database
+* AWS SSM Parameter store
```

Comment:
This might confuse developers/users because not only AWS SSM Parameter store available as Secrets Backend

Reply:
@Taragolis I am only re-using the original documentation. If you have alternative wording or suggestions, please let me know.

Reply:
It's is not strong suggestion and also possible that original docstring might not changed since Airflow 1.10.10 when secrets backend initially introduced.

BTW since airflow 2.0.0 only selected secrets backends available in the core:
- Environment variables
- Metastore database
- [Local Filesystem Secrets Backend](https://airflow.apache.org/docs/apache-airflow/stable/security/secrets/secrets-backend/local-filesystem-secrets-backend.html)

All other available only if end user install appropriate provider.